setup.py

```diff
@@ -110,14 +110,16 @@ def run(self):
 long_description = '''
 SymEngine is a standalone fast C++ symbolic manipulation library.
 Optional thin Python wrappers (SymEngine) allow easy usage from Python and
-integration with SymPy.'''
+integration with SymPy and Sage.'''
+
+exec(open('symengine/release.py').read())
```

Comment:
I followed SymPy on this, it is ugly though. Let me look at the right way to do this.
I'll fix it in both.